src/js/game/Entities/Player.js

```diff
@@ -10,6 +10,9 @@ module.exports = class Player extends BaseEntity {
     this.inventory = {};
     this.movementState = -1;
     this.onTracks = false;
+    this.getOffTrack = false;
+    this.firstTrack = true;
+    this.railDelayCheck = false;
```

Comment:
`firstTrack` and `railsDelayCheck` appear to be unused

Reply:
Good call. artifacts from when I was trying to get the right behavior. 👍